pkg/cache/queue/cluster_queue.go

```diff
@@ -237,26 +238,29 @@ func (c *ClusterQueue) backoffWaitingTimeExpired(wInfo *workload.Info) bool {
 }
 
 // Delete removes the workload from ClusterQueue.
-func (c *ClusterQueue) Delete(w *kueue.Workload) {
+func (c *ClusterQueue) Delete(log logr.Logger, w *kueue.Workload) {
 	c.rwm.Lock()
 	defer c.rwm.Unlock()
-	c.delete(w)
+	c.delete(log, w)
 }
 
 // delete removes the workload from ClusterQueue without lock.
-func (c *ClusterQueue) delete(w *kueue.Workload) {
+func (c *ClusterQueue) delete(log logr.Logger, w *kueue.Workload) {
 	key := workload.Key(w)
 	delete(c.inadmissibleWorkloads, key)
 	c.heap.Delete(key)
 	c.forgetInflightByKey(key)
 	if c.sw.matches(key) {
+		if logV := log.V(5); logV.Enabled() {
+			logV.Info("Clearing sticky workload due to deletion", "clusterQueue", c.name, "workload", key)
```

Comment:
```suggestion
			logV.Info("Clearing sticky workload", "clusterQueue", c.name, "workload", key)
```
nit, but it feels that "due to deletion" may suggest to admins / oncallers the workload was deleted completely, while this only means deletion from the queue. This can happen for example on admission